docs/reference.rst

```diff
@@ -22,7 +22,7 @@ Reference / API
    :headings: ^#
    :no-inheritance-diagram:
 
-.. automodapi:: dkist.io.loaders
+.. automodapi:: dkist.io.dask.loaders
```

Comment:
This should probably be all of the new `.dask` submodule. (Either all the sub-submodules or we need to export stuff up into the `io.dask` namespace).